source/IOSResolver/src/IOSResolverSettingsDialog.cs

```diff
@@ -248,6 +262,33 @@ private class Settings {
                 }
             }
 
+            if (settings.podfileAddUseFrameworks) {
+                GUILayout.BeginHorizontal();
+                GUILayout.Label("Enable Swift Framework Support Workaround",
+                    EditorStyles.boldLabel);
+                settings.swiftFrameworkSupportWorkaroundEnabled =
+                    EditorGUILayout.Toggle(settings.swiftFrameworkSupportWorkaroundEnabled);
+                GUILayout.EndHorizontal();
+                GUILayout.Label("This workround patches the Xcode project to properly link Swift " +
+                                "Standard Library when some plugins depend on Swift Framework " +
+                                "pods by:");
+                GUILayout.Label("1. Add a dummy Swift file to Xcode project.");
```

Comment:
just to clarify - even when ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES is set the dummy swift file is still necessary?

Reply:
Yes. :(